spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -997,7 +1003,7 @@ describe("RustCrypto", () => {
     });
 
     it("should wait for a keys/query before returning devices", async () => {
-        jest.useFakeTimers();
+        jest.useFakeTimers({ doNotFake: ["queueMicrotask"] });
```

Comment:
matrix-sdk-crypto-wasm 4.0.1 includes an update to wasm-bindgen-futures, which includes [this PR](https://github.com/rustwasm/wasm-bindgen/pull/3611), which requires that we have a working `queueMicrotask`.

In retrospect, bumping the transitive dependencies of matrix-sdk-crypto-wasm in a patch release was a mistake.